src/devices/MotorHat/DCMotor3Pwm.cs

```diff
@@ -33,7 +33,7 @@ public override double Speed
             set
             {
                 // Make sure the speed is between -1 and 1
-                _speed = Math.Clamp(value, -1, 1);
+                _speed = value < -1 ? -1 : (value > 1) ? 1 : value;
```

Comment:
Why not use the Extension you added?

Reply:
I sometimes just recoded it, to avoid having to add the extra project reference everyhwere. Either way is fine for me.